hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/TestElasticByteBufferPool.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.hadoop.io;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+public class TestElasticByteBufferPool {
```

Comment:
You are just testing the WeakReferencedElasticByteBufferPool here as well with just using the abstract base class. I think this is just duplicate and you can just change all the instances of  base class in the TestWeakReferencedElasticByteBufferPool